babelizer/data/{{cookiecutter.package_name}}/README.rst

```diff
@@ -27,80 +27,143 @@
 {%- endif %}
 
 
-{{ cookiecutter.info.project_short_description }}
+.. start-intro
+
+This project provides a wrapped version (using the `babelizer <https://babelizer.readthedocs.io>`_ tool)
+of components within the following following libraries that expose a Basic Model Interface.
+This allows these components to be imported and used within
+Python and the Python Modeling Toolkit, PyMT.
+
+.. list-table::
+  :header-rows: 1
+  :width: 90%
+  :widths: auto
+
+  * - Library
+    - Component
+    - PyMT
+  {% for babelized_class, component in cookiecutter.components|dictsort -%}
+  * - {{ component.library }}
+    - :class:`~{{ cookiecutter.package_name }}.{{ babelized_class }}`
+    -
+      .. code-block:: pycon
+
+        >>> from pymt.models import {{ babelized_class }}
+  {% endfor %}
+
+.. end-intro
+
 
 {% if is_open_source %}
 * Free software: {{ cookiecutter.open_source_license }}
 * Documentation: https://{{ cookiecutter.package_name | replace("_", "-") }}.readthedocs.io.
 {% endif %}
 
-{% set mwidth = ["Component" | length] -%}
-{%- for babelized_class, component in cookiecutter.components|dictsort %}
-    {%- if babelized_class|length > mwidth[0] -%}
-        {% set _ = mwidth.pop() -%}
-        {% set _ = mwidth.append(babelized_class|length) -%}
-    {%- endif -%}
-{%- endfor %}
-{% set max_width = mwidth[0] %}
-
-{%- set width_col_1 = max_width -%}
-{%- set width_col_2 = max_width + "`from pymt.models import `" | length -%}
-{% set fmt = "%-" + max_width|string + "s" %}
-{{ '=' * max_width }} {{ '=' * width_col_2 }}
-{{ fmt | format("Component",) }} PyMT
-{{ '=' * max_width }} {{ '=' * width_col_2 }}
+
+Quickstart
+==========
+
+.. start-quickstart
+
+To get started you will need to install the *{{ cookiecutter.package_name }}* package, which is currently distributed
+on *conda-forge*. The easiest way to install *{{ cookiecutter.package_name }}* into your current environment using either *mamba* or *conda*.
+
+.. tab:: mamba
+
+  .. code:: bash
+
+    mamba install {{ cookiecutter.package_name }}
+
+.. tab:: conda
+
+  .. code:: bash
+
+    conda install {{ cookiecutter.package_name }}
```

Comment:
Should `-c conda-forge` be explicitly added to these install instructions? I can't remember if we have instructions to add conda-forge as a default channel elsewhere.

Reply:
I'm not certain. *mamba* may use the *conda-forge* channel be default but I don't think *conda* does. I guess we should double check and possibly add the `-c conda-forge` option to these commands.